database/migrations/default/1742417174000_crash_ems_link/up.sql

```diff
@@ -0,0 +1,391 @@
+--
+-- rename fk column to crash_pk and add fk constraint
+--
+alter table ems__incidents rename column crash_id to crash_pk;
+alter table ems__incidents
+add constraint ems__incidents_crashes_crash_pk_fkey foreign key (crash_pk)
+    references public.crashes (id) on update cascade on delete set null;
+
+comment on column ems__incidents.crash_pk is 'Crash ID matched to this record';
+
+--
+-- add people fk column
+--
+alter table ems__incidents
+    add column person_id integer unique deferrable initially deferred,
```

Comment:
what is this `deferrable initially deferred` do here?

Reply:
deferred constraints are not checked until the end of a transaction.  This makes it possible to, for example, swap the `person_id` values of two records without bumping into the unique constraint. I believe we have a similar setup related to `prsn_nbr` and `unit_nbr`.